DropInDemo/Demo Base/Settings/BraintreeDemoSettings.h

```diff
@@ -18,9 +18,8 @@ typedef NS_ENUM(NSInteger, BraintreeDemoTransactionServiceThreeDSecureRequiredSt
 };
 
 typedef NS_ENUM(NSInteger, BraintreeDemoTransactionServiceThreeDSecureRequestedVersion) {
-    BraintreeDemoTransactionServiceThreeDSecureRequestedVersionOff = 0,
-    BraintreeDemoTransactionServiceThreeDSecureRequestedVersionLegacy = 1,
-    BraintreeDemoTransactionServiceThreeDSecureRequestedVersion2 = 2,
+    BraintreeDemoTransactionServiceThreeDSecureRequestedVersionLegacy = 0,
+    BraintreeDemoTransactionServiceThreeDSecureRequestedVersion2 = 1,
```

Comment:
Why did we remove the "off" feature of 3DS in the demo app? It seems like we would still want to be able to use the drop-in demo app without 3DS flows at all.

Reply:
The "off" flow would just be the same as just using a non-3DS card with the required status as "Default" or "Optional".

Reply:
So in general I think the "3DS Required Option" in the settings app currently is very confusing. The name doesn't really describe well what it does. To my understanding, that setting changes a merchant GW setting to either require (or not) 3DS transactions. If you guys have time, I would re-word these labels, but also understand if there isn't time now!

And yes, using a non-3DS card will trigger a "non 3DS" flow, but it will create a`ThreeDSecureRequest` (yes, it will fall out of this flow). Do we want an option of using the demo app that solely creates a `DropInRequest` and doesn't initialize any 3DS logic?